clock/scheduledJobs/jobCleanup.js

```diff
@@ -16,40 +16,110 @@ const Promise = require('bluebird');
 const kue = require('kue');
 const jobSetup = require('../../jobQueue/setup');
 const jobQueue = jobSetup.jobQueue;
+const conf = require('../../config');
 const rangeByStateAsync = Promise.promisify(kue.Job.rangeByState);
 kue.Job.prototype.removeAsync = Promise.promisify(kue.Job.prototype.remove);
 
 /**
  * Execute the call to clean up jobs
  *
+ * @param {Number} window - the number of jobs to delete in each batch
```

Comment:
Sorry to be a pest, @jgraff2, but can you change these variable names, too, please?